### PR TITLE
adjusted share-detail into share shell with comment component

### PR DESCRIPTION
**Pull request by etamity**

### CODE REVIEW

File: src/elements/kw-share-detail/kw-share-detail.html [hunk 1]
```diff
@@ -1,50 +1,168 @@
...
+								<p>[[share.description]]</p>
+							</div>
+							<div class="actions">
+								<button class="orange-button remix-button">remix<img src="/assets/icons/shares/remix.svg" /></button>
+							</div>
+						</div>
+						<div class="line"></div>
+						<kw-share-stats likes="[[share.likes.length]]" comments="[[share.comments_count]]"></kw-share-stats>
+					</div>
+					<kw-social-comment share=[[share]]></kw-social-comment>
```
Comment (vikdoro) [inline src/elements/kw-share-detail/kw-share-detail.html]: speech marks missing from around the share attribute value

File: src/elements/kw-share-detail/kw-share-detail.html [hunk 2]
```diff
@@ -1,50 +1,168 @@
...
-        <div id="share" class="details">
-            <iron-image src="[[share.cover_url]]" sizing="cover" hidden$="[[detailsLoaded]]"></iron-image>
-            <div id="share-container"></div>
-        </div>
-    </template>
-    <script>
+	<template>
+		<style>
+			:host {
+				display: block;
```
Comment (vikdoro) [inline src/elements/kw-share-detail/kw-share-detail.html]: my css remarks in the previous reviews would apply here as well :)

**Comment by etamity** [comment]
@vikdoro updated from code reviews

File: src/elements/kw-share-detail/kw-share-detail.html [hunk 3]
```diff
@@ -1,50 +1,167 @@
...
-        <div id="share" class="details">
-            <iron-image src="[[share.cover_url]]" sizing="cover" hidden$="[[detailsLoaded]]"></iron-image>
-            <div id="share-container"></div>
-        </div>
-    </template>
-    <script>
+	<template>
+		<style>
+			:host {
+				@apply --layout-vertical;
```
Comment (arifshanji) [inline src/elements/kw-share-detail/kw-share-detail.html]: 4 spaces please

**Review by arifshanji — CHANGES_REQUESTED** [review]

**Comment by etamity** [comment]
@arifshanji @pazdera the style has been updated

**Review by arifshanji — APPROVED** [review]

**Review by paulvarache — COMMENTED** [review]
Still issues with indentation.
The remix action is not implemented

**Comment by etamity** [comment]
@paulvarache Thanks for the review 🥇 :)

File: src/elements/kw-share-detail/kw-share-detail.html [hunk 4]
```diff
@@ -182,7 +182,8 @@
                 return this;
             },
             _onTapRemix() {
-                this.fire("remix-action");
+                var pathname = window.location.pathname.replace('/shares/','/remix/');
```
Comment (paulvarache) [inline src/elements/kw-share-detail/kw-share-detail.html]: Not every share is a Kano Code share

**Pull request merged**